client/js/widgets/scoreboard.js

```diff
@@ -0,0 +1,297 @@
+import { $, $a, removeFromDOM, asArray, escapeID } from '../domhelpers.js';
+import { Widget } from './widget.js';
+
+class Scoreboard extends Widget {
+  constructor(object, surface) {
+    super(object, surface);
+
+    this.addDefaults({
+      movable: true,
+      width: 300,
+      height: 200,
+      layer: -1,
+      typeClasses: 'widget scoreboard',
+      usePlayerColors: true,
+      playersInColumns: true,
+      rounds: null,
+      sortField: 'index',
+      sortAscending: true,
+      scoreProperty: 'score',
+      includeAllSeats: false,
+      showTotals: true,
+      seats: null,
+      showAllRounds: false,
```

Comment:
I don't understand why use different naming structures for Seats and for rounds.
it actually made me think there was no "showAllRounds" because it was not displayed next to "includeAllSeats"

Reply:
So change "includeAllSeats" to "showAllSeats"?  That way all 3 of the "show" things are in the same area and have a similar phraseology.  That makes sense to me.

Reply:
I'll change `includeAllSeats` -> `showAllSeats` (and also `usePlayerColors` -> `showPlayerColors`). Good idea.

Reply:
Done.